### ISSUE TITLE
Legend of Zelda CPU Derailed Error

### ISSUE THREAD

**Issue by KMFDManic** [Myself086/Project-Nested#26]
Hello there, fascinating work with Nested!  I have successfully converted and utilized several games.  I am currently attempting to do Zelda 1 (MSU-1).  But, it generates a CPU Derailed Error:

I used the recommended settings:

-	Recompile PRG RAM: enabled
-	NMI mode at vblank: disabled
-	Sprite zero offset: 24
-	Stack emulation, safe TSX: disabled
 
Thanks in advance for any input!

![Zelda_Error](https://user-images.githubusercontent.com/25908399/131706924-feb02f75-da4d-4030-bed9-550b3ea6c42f.png)

**Comment by Myself086** [comment]
Is this related to Conn's work on Zelda 1 MSU-1?

What did you do to get this error?

**Comment by KMFDManic** [comment]
@Myself086 That is correct.  I used nested 1.4.1 along with the readme instructions that came with the MSU-1 patch, used the recommended selected options detailed above, saved as SNES file.  I received the error with, as well as with the ips patch.  Maybe, some specific other setting/s are needed, or a break occurred since earlier nested revisions? Or, something that I did on my end.  Would certainly love to get this working.  And, I also do have a YouTube channel, so will for sure showcase the amazing applications nested has to offer!

Since Mapper 4 is most common, I feel the nintendo Canoe emulator may also be able to potentially handle nested conversions.  I will verify this theory, too

**Comment by Myself086** [comment]
Make sure to load the correct profile into Project_Nested.exe. There's one for the original release as well as RevA. The profile contains the correct settings and a patch for the NES file. Save SNES then apply the IPS patach to the SMC file.
